app-emulation/crun/crun-0.20.1.ebuild

```diff
@@ -0,0 +1,61 @@
+# Copyright 2019-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_{7..9} )
```

Comment:
Does 10 work? Usually does in simple python-any-r1 situation.

Can remove 7 too, it can't be used anymore.